modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/KubernetesPovisioningConstants.java

```diff
@@ -35,9 +35,13 @@
     public static final String KIND_DEPLOYMENT = "Deployment";
     public static final String KIND_INGRESS = "Ingress";
     public static final String KIND_SERVICE = "Service";
+    public static final String KIND_SECRETS = "Secret";
```

Comment:
SECRETS --> SECRET no S
And shouldnt it be KIND_NAME_SECRET which is more meaningful IMO. We can change other constant names as well.